File: api-java/src/main/java/fr/huiitre/tools/modules/riot/valorant/application/catalog/view/ValorantContentTierView.java
package fr.huiitre.tools.modules.riot.valorant.application.catalog.view;

import java.util.UUID;

public record ValorantContentTierView(
        Long id,
        UUID assetId,
        String name,
        String devName,
        int rank,
        int juiceValue,
        int juiceCost,
        String highlightColor,
        String displayIconUrl) {}

File: api/Modules/Riot/Valorant/Application/Catalog/Views/ValorantContentTierView.cs
namespace Tools.Api.Modules.Riot.Valorant.Application.Catalog.Views;

public sealed record ValorantContentTierView(
    long Id,
    Guid AssetId,
    string Name,
    string DevName,
    int Rank,
    int JuiceValue,
    int JuiceCost,
    string? HighlightColor,
    string? DisplayIconUrl
);
